modin/pandas/test/test_io.py

```diff
@@ -1893,6 +1893,16 @@ def test_to_gbq():
         modin_df.to_gbq("modin.table")
 
 
+def test_internal_error_2322():
```

Comment:
I think, that name of test should be more clearly to reflect what it is about without visiting github.

Reply:
Changed.